### PR TITLE
docs: Fix markdown

### LINKED ISSUE #278: Fix markdown
Resolve issues in the markdown files in this repo.

### PR DESCRIPTION
**Pull request by jodh-intel**
Resolve issues with markdown in this repo.

Fixes #278.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: rootfs-builder/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-* [Creating a rootfs with kernel modules](#creating-a-rootfs-with-kenrel-modules)
-* [Build a rootfs using Docker*](#build-a-rootfs-using-docker*)
+* [Creating a rootfs with kernel modules](#creating-a-rootfs-with-kernel-modules)
+* [Build a rootfs using Docker*](#build-a-rootfs-using-docker)
 * [Adding support for a new guest OS](#adding-support-for-a-new-guest-os)
     * [Create template files](#create-template-files)
     * [Modify template files](#modify-template-files)
     * [Expected rootfs directory content](#expected-rootfs-directory-content)
-    * [(optional) Customise the rootfs](#(optional)-customise-the-rootfs)
+    * [optional - Customise the rootfs](#optional-customise-the-rootfs)
```
Comment (marcov) [inline rootfs-builder/README.md]: Do you mind capitalizing "optional" (here and below)?

Comment (jodh-intel) [reply rootfs-builder/README.md]: Done.

**Comment by jodh-intel** [comment]
Ping y'all - we need these doc changes to land ;)

**Comment by grahamwhaley** [comment]
@gabibeyer - you are my review buddy this week I think - ptal? :-)

**Comment by jodh-intel** [comment]
Just a reminder to all that doc PRs are arguably the *most* important type of PRs too :)

- https://github.com/kata-containers/community/blob/master/Documentation-Review-Process.md#users-should-always-expect-accurate-documentation

We could get fancy and auto-label doc-only PRs in future I think.

**Comment by jodh-intel** [comment]
Thanks @chavafg.

**Pull request merged**